### ISSUE TITLE
Images in dark mode

### ISSUE THREAD

**Issue by christianp** [numbas/Numbas#1148]
I think that static images should have a white background under them, because I don't think it's reasonable to ask authors to prepare two versions of each image.

Interactive diagrams such as those produced by JSXGraph and Eukleides could maybe follow the theme's colours, or at least use different colours in dark mode, and explicitly set a background colour.
